pkg/k8s/endpoints.go

```diff
@@ -253,6 +258,9 @@ func ParseEndpointSliceV1Beta1(ep *slim_discovery_v1beta1.EndpointSlice) *Endpoi
 						metrics.TerminatingEndpointsEvents.Inc()
 					}
 				}
+				if zoneName, ok := sub.Topology["topology.kubernetes.io/zone"]; ok {
```

Comment:
nit: could use `k8s.io/api/core/v1.LabelTopologyZone` here instead of the verbatim string `topology.kubernetes.io/zone`

Reply:
In this file `github.com/cilium/cilium/pkg/k8s/slim/k8s/api/core/v1` is used. Is it ok to import both or should I replace slim version with the regular one?

Reply:
Thanks for quick reply. Done.

I also replaced hostname label reference with const for consistency in this file